Day-I-Social-Card/social-card/src/components/HeaderComponents/HeaderTitle.js

```diff
@@ -1,2 +1,12 @@
 import React from 'react';
 import './Header.css';
+const HeaderTitle=()=>{
+    return(
+        <div>
+            <h1>Lambda School</h1>
+            <p>@LambdaSchool</p>
+            <p>Date.now</p>
```

Comment:
So with JSX when you want to embed so JS in HTML you have to to use `{}` so for this you would be `<p>{Date.now()}</p>`

Reply:
yeah i was getting to that later because i cant get anything to work i havent looked at your first comment i need to rewatch the video but ill get it
typing up the standup form now